RoosterPlanner.UI/src/app/pages/project/project.component.scss

```diff
@@ -1,12 +1,16 @@
-* {
+img {
+  width: 70%;
+  height: auto;
+}
+
+h1 {
   text-align: left;
   font: normal normal normal 35px/50px Segoe UI;
```

Comment:
Kan font niet in een algemenere .scss file (hoger liggend, omdat dit waarschijnlijk voor alle h1's hetzelfde is)

Reply:
Helemaal gelijk. Ik neem dit mee naar volgende sprint daarin ben ik van plan een CSS refactor slag te maken

Reply:
Ok. Check bij die slag dan ook of je em kan gebruiken.